gradle-plugin-testing-error-prone/src/main/java/com/palantir/gradle/testing/errorprone/GradlePluginTestHelpers.java

```diff
@@ -37,13 +43,56 @@ static boolean notWithinGradlePluginTests(Tree tree, VisitorState state) {
         return !WITHIN_GRADLE_PLUGIN_TESTS_CLASS.matches(tree, state);
     }
 
-    static boolean isGradlePluginTestsLibraryMethod(MethodInvocationTree tree, VisitorState state) {
+    static boolean notGradlePluginTestsLibraryMethod(MethodInvocationTree tree) {
         return Optional.ofNullable(ASTHelpers.getSymbol(tree))
                 .map(Symbol.MethodSymbol::enclClass)
                 .map(classSymbol -> classSymbol.getQualifiedName().toString())
                 .filter(className -> className.startsWith(LIBRARY_PACKAGE))
+                .isEmpty();
```

Comment:
it's useful to not in the PR when a bunch of methods have just been moved from one file to another, or, as is the case for findVariableInitializer I think, extracted from another method in another file